fe/fe-core/src/test/java/com/starrocks/service/FrontendServiceImplTest.java

```diff
@@ -956,4 +978,46 @@ public void testLoadTxnCommitFailed() throws UserException, TException {
         TLoadTxnCommitResult result = impl.loadTxnCommit(request);
         Assert.assertEquals(TStatusCode.ANALYSIS_ERROR, result.status.status_code);
     }
+
+    @Test
+    public void testAddListPartitionConcurrency() throws UserException, TException {
```

Comment:
Why not test this feature with sql-test?

Reply:
sql-test cannot simulate this precise concurrency scenario, because it actually has double detection, and generally cannot reach this part of the code, so even if it is run many times, the coverage is 0